docs/openj9_newuser.md

```diff
@@ -0,0 +1,100 @@
+
+- The system the VM is running on and the resources available.
+- The Java execution environment, including the options set from the command line.
+- The native memory used by the VM, broken down by VM component.
+- Memory usage in the VM for the object heap and internal VM structures, such as the JIT code cache.
+- Lock operations that protect shared resources during runtime.
+- Java threads, native threads, and stack traces.
+- Hook interfaces, for performance analysis.
+- Details about the shared classes cache, if used.
+- Detailed information about classloaders, together with a list of libraries and classes that are loaded.
+
+For more information, see [Java dump](dump_javadump.md).
+
+### Tools
+
+OpenJ9 provides support for a number of monitoring and diagnostic tools that can be found in the [Eclipse marketplace](https://marketplace.eclipse.org/). Each tool provides a graphical user interface to help you visualize data and, in some cases, can provide tuning or debugging recommendations.
+
+- [**Health Center:**](https://marketplace.eclipse.org/content/ibm-monitoring-and-diagnostic-tools-health-center) Provides real-time monitoring of running applications with minimal overhead over the network. You can monitor a whole range of operations including, class loading, CPU usage, GC heap and pause times, I/O activity, lock contention, method trace, native memory usage, profiling, and live threads. For more information, read the [Health Center documentation](https://www.ibm.com/support/knowledgecenter/en/SS3KLZ/com.ibm.java.diagnostics.healthcenter.doc/homepage/plugin-homepage-hc.html).
+- [**Garbage Collection Memory Vizualizer (GCMV):**](https://marketplace.eclipse.org/content/ibm-monitoring-and-diagnostic-tools-garbage-collection-and-memory-visualizer-gcmv) Plots GC and native memory data over time. You can view and save data as a report, raw log, tabulated data, or in graphical format. The tool helps to diagnose problems such as memory leaks with data presented in various visual formats for analysis. Tuning recommendations are also provided. For more information, read the [GCMV documentation](https://www.ibm.com/support/knowledgecenter/en/SS3KLZ/com.ibm.java.diagnostics.visualizer.doc/homepage/plugin-homepage-gcmv.html).
+- [**Memory Analyzer:**](https://marketplace.eclipse.org/content/memory-analyzer-0) Examines the Java object heap to help find memory leaks or reduce memory consumption. Support is available for OpenJ9 via the DTFJ interface. More information about Eclipse MAT can be found on the [project website page](https://www.eclipse.org/mat/).
```

Comment:
We should give instructions on how to load the DTFJ plugin into MAT. Afaik this information is only available in one of the release notes. https://www.eclipse.org/openj9/docs/version0.9/ `installing the DTFJ plugin. (Install from the Eclipse Help menu; Install New Software > Work with "IBM Diagnostic Tool Framework for Java" > IBM Monitoring and Diagnostic Tools > Diagnostic Tool Framework for Java)`

Reply:
Done